localisation/00_wc_generics.csv

```diff
@@ -568,3 +568,59 @@ wc_tribalmunitions1_desc;Herbal Healer;;Kr
 wc_tribalmunitions2_desc;Tribal Outposts;;Stammes-Au�enposten;;;;;;;;;;;x
 wc_tribalmunitions3_desc;Winter Shelters;;Winterunterk�nfte;;;;;;;;;;;x
 wc_tribalmunitions4_desc;Barbaric Silo;;Barbarensilo;;;;;;;;;;;x
+;;;;;;;;;;;;;;
+wc_merchantVenue;Merchant Venue;;;;;;;;;;;;;x
```

Comment:
It's probably better to move outdated loc, it's not good thing to increase or translators workload with things than got deleted